power/battery_linux.go

```diff
@@ -0,0 +1,118 @@
+// +build linux
```

Comment:
...and instead compile this module on Linux, which has exactly the same names, but different implementations? If so, that's very sweet.

Reply:
At first, `switch runtime.GOOS`, but there were many switch-case clauses. So I created files for Linux